esphome/esp32-c3-mini--goe11-controll.yaml

```
@@ -78,9 +78,9 @@ time:
 sun:
   latitude: !secret latitude
   longitude: !secret longitude
-  on_sunset: 
-    then: 
-      - switch.turn_off: enable_pv_charging
+  #on_sunset: 
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)